api/app/tests/requests/test_pteams.py

```diff
@@ -2980,121 +2981,93 @@ def test_it_should_set_requester_if_assignee_is_not_specify_and_saved_current_us
+            content_fingerprint="dummy_fingerprint",
+        )
+        persistence.create_vuln(testdb, self.vuln1)
 
-    def _set_ticket_status(
-        self, pteam_id: str, service_id: str, ticket_id: str, request: dict
-    ) -> dict:
-        url = f"/pteams/{pteam_id}/services/{service_id}/ticketstatus/{ticket_id}"
-        user1_access_token = self._get_access_token(USER1)
-        _headers = {
-            "Authorization": f"Bearer {user1_access_token}",
-            "Content-Type": "application/json",
-            "accept": "application/json",
-        }
-        return client.put(url, headers=_headers, json=request).json()
+        affect1 = models.Affect(
+            vuln_id=self.vuln1.vuln_id,
+            package_id=self.package1.package_id,
+            affected_versions=["<=1.0.0"],
+            fixed_versions=["2.0.0"],
+        )
+        persistence.create_affect(testdb, affect1)
 
-    def test_returns_empty_if_no_tickets(self, not_actionable_topic1):
-        url = (
-            f"/pteams/{self.pteam1.pteam_id}/services/{self.service_id1}"
-            f"/topics/{self.topic1.topic_id}/tags/{self.tag1.tag_id}/tickets"
+        self.threat1 = models.Threat(
+            package_version_id=self.package_version1.package_version_id, vuln_id=self.vuln1.vuln_id
         )
-        user1_access_token = self._get_access_token(USER1)
-        _headers = {
-            "Authorization": f"Bearer {user1_access_token}",
-            "Content-Type": "application/json",
-            "accept": "application/json",
-        }
-        response = client.get(url, headers=_headers)
-        assert response.status_code == 200
-        assert response.json() == []
+        persistence.create_threat(testdb, self.threat1)
 
-    def test_returns_ticket_with_initial_status_if_no_status_created(
-        self, testdb, actionable_topic1
-    ):
-        db_dependency1 = testdb.scalars(select(models.Dependency)).one()
-        db_threat1 = testdb.scalars(select(models.Threat)).one()
+        ticket_business.fix_ticket_by_threat(testdb, self.threat1)
+
+    def test_it_should_return_200_when_ticket_exists(self, testdb):
+        # Given
         db_ticket1 = testdb.scalars(select(models.Ticket)).one()
         db_status1 = testdb.scalars(select(models.TicketStatus)).one()
         expected_ticket_response1 = {
             "ticket_id": str(db_ticket1.ticket_id),
-            "threat_id": str(db_threat1.threat_id),
+            "threat_id": str(self.threat1.threat_id),
             "created_at": datetime.isoformat(db_ticket1.created_at),
             "ssvc_deployer_priority": (
                 None
                 if db_ticket1.ssvc_deployer_priority is None
                 else db_ticket1.ssvc_deployer_priority.value
             ),
+            "ticket_safety_impact": (
+                None
+                if db_ticket1.ticket_safety_impact is None
+                else db_ticket1.ticket_safety_impact.value
+            ),
```

Comment:
下記で問題ないように見えます。ssvc_deployer_priorityも同様
```suggestion
            "ticket_safety_impact": db_ticket1.ticket_safety_impact.value,
```

Reply:
db_ticket1.ticket_safety_impactがNoneの時に、db_ticket1.ticket_safety_impact.valueのようにvalueまで指定するとエラーが出ます。そのため、三項演算子で条件分岐させています。